pkg/tag/tag.go

```diff
@@ -146,6 +149,8 @@ func GetVRKind(tag Tag, vr string) VRKind {
 		return VRFloat64List
 	case "SQ":
 		return VRSequence
+	case "UN":
+		return VRUnknown
```

Comment:
just a note/thought to myself...wondering if we need to revisit the point of having VRKind construct. There's new behavior now, with VRUnknown, the value could actually be represented in two different containers: as a sequence or as bytes (which is new, at least given the behaviors of other items in the list). And anyway, it's not like we need VRKind to unpack the value stored in the container (we could use ValueType for that). So basically the VRKind construct at the moment just serves to "group" some VRs together (e.g. OW and OB) into the type of value _we_ unpack it as in our program. But in reality, we don't need this and could just switch on the VRs directly in readValue or possibly group raw VRs by ValueType we wish to represent them as internally (but again, this doesn't work for UN, because we need other information to determine if we'll store it as bytesValue or sequenceValue). 

Anyway, just some quick off the top of my head thoughts for me to revisit later.